index.js

```diff
@@ -68,7 +68,52 @@ function toggleColor(element) {
   }
 }
 
+const header = document.querySelector('#header')
+
+header.addEventListener('click', function() {
+  toggleColor(header)
+})
+
+// header.addEventListener('click', toggleColor)
+// why doesn't this work?
+
 /***** Deliverable 2 *****/
 
+const likeBtn = document.querySelector('.like-button')
+const pLike = document.querySelector('.likes')
+
+// likeBtn.addEventListener('click', function(){
+//   const likes = parseInt(pLike.textContent) + 1 + " Likes"
+//   pLike.textContent = likes
+// })
+
+likeBtn.addEventListener('click', function(){
+  const likes = parseInt(pLike.textContent)
+  pLike.textContent = likes + 1 + " Likes"
```

Comment:
Looks good! The key here is that you have to read some information from the DOM to get the current number of likes, increment it, and then write it back to the DOM. You can also do some string interpolation here (just a matter of preference): 

```js
likes.textContent = `${totalLike + 1} Likes`
```